src/NexusMods.App.UI/Pages/Settings/SettingsPageViewModel.cs

```diff
@@ -49,6 +46,21 @@ public SettingsPageViewModel(ISettingsManager settingsManager, IWindowManager wi
             }
         }, this.WhenAnyValue(vm => vm.HasAnyValueChanged));
 
+        CancelCommand = ReactiveCommand.Create(() =>
+        {
+            // TODO: ask to discard current values
+            foreach (var viewModel in SettingEntries)
+            {
+                viewModel.InteractionControlViewModel.ValueContainer.ResetToPrevious();
+            }
+        }, this.WhenAnyValue(vm => vm.HasAnyValueChanged));
+
+        CloseCommand = ReactiveCommand.Create(() =>
+        {
+            // TODO: check if any values have changed and ask for confirmation to close
+            GetWorkspaceController().ClosePanel(WorkspaceId, PanelId);
```

Comment:
This crashes the app if it is the last panel. I would just hide the X button for now. We can show it back when we put this in a Popped out panel.

Reply:
Doesn't crash the app on my end. You can put this in the workspace refinement issue.

Reply:
Lausandy said we should remove/hide the X button. And the title page. That should solve this.